alea/examples/configs/unbinned_wimp_statistical_model.yaml

```diff
@@ -1,7 +1,6 @@
 parameter_definition:
   wimp_mass:
     nominal_value: 50
-    ptype: shape
     fittable: false
     description: WIMP mass in GeV/c^2
     blueice_anchors:
```

Comment:
The anchors here should be removed. We should also raise an error if anchors are provided for `needs_reinit` parameter.

Reply:
removed -- good point. Not sure if we need to raise an error though. Maybe rather a warning if you pass anchors for a parameter that's not a shape? Because doing this won't hurt anyone it's just useless, right?

Reply:
Yes, you are right. My point is only to be strict. Maybe just a warning, because anyway we raise error in `fit` and `get_expectation_values` if another value is assigned.